modules/ding_gatewayf/ding_gatewayf.info

```diff
@@ -0,0 +1,6 @@
+name = Ding gatewayf
+description = Support for login at gatewayf.
```

Comment:
I don't have any information about gatewayf... It's something DBC have invented. The only this I have is an mail with the end-point and the 3 parameters it takes in the URL.